app/models/user.rb

```diff
@@ -1,8 +1,16 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] It would be nice if you could add the `:confirmable` so emails are confirmed when users are registered before logging in. Kindly add that as it's required to have in your application 👍🏽

<img width="974" alt="Screenshot 2024-02-01 at 9 45 49 AM" src="https://github.com/Eccb7/Pi-Blogs/assets/50941074/1ac8036f-cb21-4a78-97fd-4c9f7434856e">